libs/brig-types/src/Brig/Types/Team/Invitation.hs

```diff
@@ -7,19 +7,22 @@ import Brig.Types.Common
 import Data.Aeson
 import Data.Id
 import Data.Json.Util
+import Galley.Types.Teams (Role)
```

Comment:
Let's please not make a service types package depend on another service types package; it's just a matter of time until `galley-types` depends on `brig-types` and then we're stuck with circular dependencies.

If we need to share types, let's add them to the types-common package (FWIW, I see that `galley-types` currently depends on `gundeck-types` and we should aim to get rid of that in the same fashion).